### PR TITLE
Add nginx conf and dockerize client

### PR DESCRIPTION
**Pull request by kevinahuber**
This adds a production build pipeline and nginx server as part of #7

This does not address wiring the application client & server up- I have a few questions and thoughts around doing this, especially since the client prod vs dev differs greatly. 

1) Generally, what is the end expected deliverable? Is this PR overreaching, or do we want a nicely buttoned up production app? 
2) @carrollgt91 as someone with Docker chops- do you have strong opinions on how to deviate dev vs. prod Dockerfile & docker-compose.yml? How about doing:
```
# Prod
Dockerfile
docker-compose.yml

#Dev 
dev.Dockerfile
dev.docker-compose.yml

#Other env (testing, etc)
env.Dockerfile
env.docker-compose.yml
```
I'm a bit rusty in this category, and quick googling doesn't show a great coalescing.

3) Do we want to pursue hot reloading on FE & BE at the same time? This is the part I have some ideas on but could see some speed bumps. 

I'm also open to feedback on everything- approach, technologies used, etc.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+    image: openmind/pis-client:latest
+    build: ./client/
+    ports:
+      - 80:80
   server:
-    image: openmined/pis:latest
+    image: openmined/pis-server:latest
     build: ./server/
-    volumes: ['./server:/server']
+    volumes: ["./server:/server"]
```
Comment (kevinahuber) [inline docker-compose.yml]: Yikes, this is from my IDE. Y'all have a preference or codestyle enforcement mechanism?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,9 +1,14 @@
-version: '3'
+version: "3"
 services:
+  client:
+    image: openmind/pis-client:latest
```
Comment (hericlesme) [inline docker-compose.yml]: openmind -> openmined

File: docker-compose.yml (hunk 1 above)
Comment (carrollgt91) [reply docker-compose.yml]: Not yet! This is fine, though :) thanks!

**Comment by carrollgt91** [comment]
Going to go ahead and merge this, as it will assist Tom in getting started on the backend integration.

Right now, we're just focusing on dev - there are a number of important questions we'll need to ask before really digging into production configuration, and it likely will be more coupled to the initial deployment environment. So, for now, this looks great! Thanks Kevin!

**Comment by carrollgt91** [comment]
Re: hot reloading both projects when one is saved - not for now, no. I think that's easy enough to work around.

**Pull request merged**